setup.py

```diff
@@ -178,7 +180,7 @@ def additional_commands():
         description='explicitly typed attributes for Python',
         long_description=open('README.rst').read(),
         download_url='https://github.com/enthought/traits',
-        install_requires=__requires__,
+        install_requires=["six"],
```

Comment:
given this change, do you want to remove the duplication in `traits.__init__`?

Reply:
Possibly. That's an actual package change rather than just an infrastructure change, so it has the potential to break code. It's *probably* safe to assume that no-one is doing `from traits import __requires__`, but it's hard to be sure.

Reply:
I've removed it.